flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -221,6 +221,14 @@ under the License.
 									<shadedPattern>org.apache.flink.kinesis.shaded.org.apache.http</shadedPattern>
 								</relocation>
 							</relocations>
+							<filters>
+								<filter>
+									<artifact>com.amazonaws:amazon-kinesis-producer</artifact>
+									<excludes>
+										<exclude>META-INF/THIRD_PARTY_NOTICES</exclude>
```

Comment:
it's so annoying that there isn't a standard location for this stuff...

Reply:
Agreed...